CHANGELOG.rst

```diff
@@ -1,6 +1,61 @@
 Changelog
 =========
 
+1.19.0 (2024-02-21)
+-------------------
+
+* Add ExternalDeploymentSensor (#1472)
+* Deprecate the following operators and sensors and proxy them to their upstream Apache Airflow providers' deferrable counterparts
+    * EmrContainerSensorAsync
+    * EmrStepSensorAsync
+    * EmrJobFlowSensorAsync
+    * RedshiftClusterSensorAsync
+    * BatchSensorAsync
+    * S3KeySensorAsync
+    * S3KeysUnchangedSensorAsync
+    * EmrContainerOperatorAsync
+    * RedshiftDeleteClusterOperatorAsync
+    * RedshiftResumeClusterOperatorAsync
+    * RedshiftPauseClusterOperatorAsync
+    * BatchOperatorAsync
+    * SageMakerProcessingOperatorAsync
+    * SageMakerTransformOperatorAsync
+    * SageMakerTrainingOperatorAsync
+    * RedshiftDataOperatorAsync
+    * GCSObjectExistenceSensorAsync
+    * GCSObjectsWithPrefixExistenceSensorAsync
+    * GCSUploadSessionCompleteSensorAsync
+    * GCSObjectUpdateSensorAsync
+    * GKEStartPodOperatorAsync
+    * BigQueryInsertJobOperatorAsync
+    * BigQueryCheckOperatorAsync
+    * BigQueryGetDataOperatorAsync
+    * BigQueryIntervalCheckOperatorAsync
+    * BigQueryValueCheckOperatorAsync
+    * DataprocCreateClusterOperatorAsync
+    * DataprocDeleteClusterOperatorAsync
+    * DataprocSubmitJobOperatorAsync
+    * DataprocUpdateClusterOperatorAsync
+    * BigQueryTableExistenceSensorAsync
+    * AzureDataFactoryPipelineRunStatusSensorAsync
+    * WasbBlobSensorAsync
+    * WasbPrefixSensorAsync
+    * AzureDataFactoryRunPipelineOperatorAsync
+    * FileSensorAsync
+    * ExternalTaskSensorAsync
+    * SnowflakeSqlApiOperatorAsync
+    * DatabricksSubmitRunOperatorAsync
+    * DatabricksRunNowOperatorAsync
+    * KubernetesPodOperatorAsync
+    * SFTPSensorAsync
+    * HttpSensorAsync
+    * DbtCloudJobRunSensorAsync
+    * DbtCloudRunJobOperatorAsync
+    * LivyOperatorAsync
+* Add deprecation notice docs for astronomer-providers (#1468)
+* Increase min airflow version to 2.6 (#1438)
```

Comment:
Sorry, just remembered, let's also explicitly mention that we also bumped up min Python version to 3.8 (which however does come implicitly from Airflow 2.6)

https://github.com/astronomer/astronomer-providers/pull/1442

Reply:
Sure. Just updated it